src/openrct2/ride/Vehicle.cpp

```diff
@@ -591,140 +508,114 @@ const rct_vehicle_info *vehicle_get_move_info(sint32 cd, sint32 typeAndDirection
 
 uint16 vehicle_get_move_info_size(sint32 cd, sint32 typeAndDirection)
 {
-    if (!vehicle_move_info_valid(cd, typeAndDirection, 0)) {
+    if (!vehicle_move_info_valid(cd, typeAndDirection, 0))
+    {
         return 0;
     }
     return gTrackVehicleInfo[cd][typeAndDirection]->size;
 }
 
-static const uint8 DoorOpenSoundIds[] = {
-    SOUND_DOOR_OPEN,
-    SOUND_62
-};
+static const uint8 DoorOpenSoundIds[] = { SOUND_DOOR_OPEN, SOUND_62 };
 
-static const uint8 DoorCloseSoundIds[] = {
-    SOUND_DOOR_CLOSE,
-    SOUND_62
-};
+static const uint8 DoorCloseSoundIds[] = { SOUND_DOOR_CLOSE, SOUND_62 };
 
-static const struct { sint8 x, y, z; } SteamParticleOffsets[] = {
-    { -11,   0, 22 },
-    { -10,   4, 22 },
-    {  -8,   8, 22 },
-    {  -4,  10, 22 },
-    {   0,  11, 22 },
-    {   4,  10, 22 },
-    {   8,   8, 22 },
-    {  10,   4, 22 },
-    {  11,   0, 22 },
-    {  10,  -4, 22 },
-    {   8,  -8, 22 },
-    {   4, -10, 22 },
-    {   0, -11, 22 },
-    {  -4, -10, 22 },
-    {  -8,  -8, 22 },
-    { -10,  -4, 22 },
-    {  -9,   0, 27 },
-    {  -8,   4, 27 },
-    {  -6,   6, 27 },
-    {  -4,   8, 27 },
-    {   0,   9, 27 },
-    {   4,   8, 27 },
-    {   6,   6, 27 },
-    {   8,   4, 27 },
-    {   9,   0, 27 },
-    {   8,  -4, 27 },
-    {   6,  -6, 27 },
-    {   4,  -8, 27 },
-    {   0,  -9, 27 },
-    {  -4,  -8, 27 },
-    {  -6,  -6, 27 },
-    {  -8,  -4, 27 },
-    { -13,   0, 18 },
-    { -12,   4, 17 },
-    {  -9,   9, 17 },
-    {  -4,   8, 17 },
-    {   0,  13, 18 },
-    {   4,   8, 17 },
-    {   6,   6, 17 },
-    {   8,   4, 17 },
-    {  13,   0, 18 },
-    {   8,  -4, 17 },
-    {   6,  -6, 17 },
-    {   4,  -8, 17 },
-    {   0, -13, 18 },
-    {  -4,  -8, 17 },
-    {  -6,  -6, 17 },
-    {  -8,  -4, 17 }
-};
+static const struct
+{
+    sint8 x, y, z;
+} SteamParticleOffsets[] = { { -11, 0, 22 }, { -10, 4, 22 },  { -8, 8, 22 },  { -4, 10, 22 },  { 0, 11, 22 },  { 4, 10, 22 },
```

Comment:
I really don't the autoformatter formatting tables like this. Can't you tell it to align vertically?

Reply:
I've done the best I could to make it similar to our inconsistent style. I can add exception to the formatting here.

Reply:
Our inconsistent style seems more consistent than what Clang does. Just have the table vertical, not all over the place.